File: Day/Java/ArmstrongNumberChecker.java





public class ArmstrongNumberChecker {

public static void main(String[] args) {

int number = 153;

int originalNumber = number;

int sum = 0;

while (number != 0) {

int digit = number % 10;

sum = sum + (digit * digit * digit);

number = number / 10;
}

if (sum == originalNumber) {

System.out.println("Armstrong number");

} else {

System.out.println("Not an Armstrong number");
}
}
}

File: Day/JavaScript/ArmStrongNUmberChecker.js


let number = 153;

let originalNumber = number;

let sum = 0;

while (number !== 0) {

let digit = number % 10;

sum = sum + (digit * digit * digit);

number = Math.floor(number / 10);
}

if (sum === originalNumber) {

console.log("Armstrong number");

} else {

console.log("Not an Armstrong number");
}
